packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,372 @@
+<img src="./images/mercury-brand.svg" alt="Mercury brand" width="30" />
+
+Mercury Design System
+
+# <img src="./images/angular.svg" alt="Angular isotype" width="30" /> Installation for Angular
+
+- [1. Installation](#1-installation)
+
+- [2. Usage](#2-usage)
+
+  - [2.1. Before starting, define the path for the CSS bundles, custom fonts and icons](#21-before-starting-define-the-path-for-the-css-bundles-custom-fonts-and-icons)
+
+  - [2.2. Create the CSS bundles](#22-create-the-css-bundles)
+
+    - [2.2.1. Using the CLI to create the CSS bundles](#221-using-the-cli-to-create-the-css-bundles)
+
+    - [2.2.2. Using SASS to transpile the CSS bundles](#222-using-sass-to-transpile-the-css-bundles)
+
+    - [2.2.3. Using already generated CSS bundles](#223-using-already-generated-css-bundles)
+
+  - [2.3. Copy the CSS bundles, custom fonts and icons to the dev and prod builds](#23-copy-the-css-bundles-custom-fonts-and-icons-to-the-dev-and-prod-builds)
+
+    - [2.3.1. Copying assets](#231-copying-assets)
```

Comment:
It is not necessary to create a sub-section if only one exists. Collapse the content of the 2.3 and 2.3.1.

The same reasoning applies to all occurrences of this pattern.

```suggestion
  - [2.3. Copy the CSS bundles, custom fonts and icons to the dev and prod builds](#23-copy-the-css-bundles-custom-fonts-and-icons-to-the-dev-and-prod-builds)
```